mender/values.yaml

```diff
@@ -66,8 +66,8 @@ api_gateway:
       memory: 512M
   affinity: {}
   nodeSelector: {}
-  httpPort: 80
-  httpsPort: 443
+  httpPort: 9080
+  httpsPort: 9443
```

Comment:
please set these back to their defaults

Reply:
After a deep check I can confirm this modification works properly and you can keep it.